src/scene/shader-lib/chunks/standard/frag/litShaderArgs.js

```diff
@@ -0,0 +1,85 @@
+export default /* glsl */`
+
+struct IridescenceArgs
+{
+    // The color of the fresnel effect, set by the backend
```

Comment:
this is 'set by the backend' - should it be specified in the structure that represents the input to the backend?

Reply:
What do you mean? Pointing this out means that setting this in the frontend will have no effect as the value gets replaced sometime later.

Reply:
so why expose this value in a structure the user is expected to fill with data, if the data is ignored / overwritten? It should probably not be part of this struct which represents a public API as an output of front end.